Dockerfile

```diff
@@ -369,15 +384,15 @@ RUN curl -fsSL get.docker.com -o /tmp/get-docker.sh &&\
     sh /tmp/get-docker.sh
 
 RUN go get -u golang.org/x/lint/golint &&\
-    go get github.com/derekparker/delve/cmd/dlv &&\
+    if [ `uname -m` != "aarch64" ]; then go get github.com/derekparker/delve/cmd/dlv; fi &&\
     go get github.com/tebeka/go2xunit
 
 # Get integration test deps in here
 COPY python/setup.py /tmp/
 RUN pip3 install -e /tmp/
 COPY tests/requirements.txt /tmp/
 RUN pip3 install --upgrade pip==9.0.1 && pip3 install -r /tmp/requirements.txt
-RUN wget -O /usr/bin/gomplate https://github.com/hairyhenderson/gomplate/releases/download/v2.3.0/gomplate_linux-amd64-slim &&\
+RUN wget -O /usr/bin/gomplate https://github.com/hairyhenderson/gomplate/releases/download/2.8.0/gomplate_linux-${TARGET_ARCH}-slim &&\
```

Comment:
If you change the version of gomplate here you need to change the version in https://github.com/signalfx/signalfx-agent/blob/master/.circleci/config.yml#L200 so that the docs check in CI is consistent with what developers are using.

Reply:
Updated to match.